src/layouts/base.njk

```diff
@@ -16,7 +16,15 @@
     {% endblock %}
   </head>
   <body>
-    {% include "header.njk" %}
+    <header
+      class="Header"
+      style="background: #20123a url('{{ featuredImage }}') no-repeat center;">
```

Comment:
This will require unsafe-inline for the CSP. Is there no way to do this with an inline image?

Reply:
I don't know. I guess we'll see once we implement the final design.